spring-integration-cassandra/src/main/java/org/springframework/integration/cassandra/outbound/CassandraMessageHandler.java

```diff
@@ -175,68 +197,74 @@ protected Object handleRequestMessage(Message<?> requestMessage) {
 
 		Object result = payload;
 
-		Type queryType = this.queryType;
+		OperationType queryType = this.queryType;
```

Comment:
Sorry  but I did not know about `Spring Cloud Data Flow`, anyway you right that the change to `CassandraMessageHandler` need to be  addressed in separate jira. The goal of this issue is to support namespace handler and for now I've reverted all change on CassandraMessageHandler to  `2e9247711d72959dc1a3509451bd377e34ec1c29`